_data/projects/HoneyBot.yml

```diff
@@ -0,0 +1,10 @@
+name: HoneyBot
+desc: python powered OOP modular (plugins-supported) IRC bot. ignited in Mauritius.
+site: https://github.com/Abdur-rahmaanJ/honeybot
+tags:
+- python
+- irc
+- bot
+upforgrabs:
+  name: Help wanted
```

Comment:
@Abdur-rahmaanJ Your label name is "help wanted" as I can see. So, please change this to lower case.